app/javascript/courses/review/components/CoursesReview__ChecklistEditor.re

```diff
@@ -195,7 +195,9 @@ let make =
                   <School__InputGroupError
                     message="Title should be greater than 2 characters"
                     active={
-                      reviewChecklistItem |> ReviewChecklistItem.title == ""
+                      reviewChecklistItem
+                      |> ReviewChecklistItem.title
+                      |> String.length <= 2
```

Comment:
As you'd mentioned, we should trim this before checking the length.

However, since this very same check is also performed on `resultItem` (below), it would be cleaner to extract a function here. Maybe a `validTitle: string => bool` at the top-level of this module for use in both locations?